sous-chef/sous_chef/recipe_book/_recipe_book.py

```diff
@@ -227,9 +240,45 @@ def _get_recipe_book() -> pd.DataFrame:
         )
         dataframe["factor"] = 1
         dataframe["amount"] = None
+        # cannot pickle pint units at this time
+        dataframe["quantity"] = None
         dataframe = dataframe.replace("nan", pd.NA)
+
+        dataframe.time_preparation = dataframe.time_preparation.astype(
```

Comment:
somehow by converting from panderas to pydantic -> panderas, these steps were needed more explicitly. as we have a failing test, i'm a bit of the mind to just roll that back as it's not needed. what do you think? do you see an easy fix?